website/docs/getting-started/for-designers/partials/best-practices.md

```diff
@@ -1,71 +1,87 @@
 
-!!!
+As we work to achieve parity between Structure and Helios, use this decision tree to understand how to move forward within your project:
 
-**Note:** Much of this already exists from Design Ops, waiting to collaborate with them on the content.
+![Decision tree flow chart](/assets/getting-started/designers/hds-decision-tree.png)
 
-## File hygiene
+**Does the element (or a comparable alternative) exist in Helios?**
 
-- Use a cover photo
-- Differentiate between delivered work, and work in progress
+If so, great! Using this element ensures it will be supported in future versions of Helios and that it shares a common visual language with the rest of the elements.
 
-### Naming conventions
+**If no, does the element (or a comparable alternative) exist in Structure?**
 
-Naming things is hard, and while naming components and elements within a design system is often different from naming elements further down the UX funnel, the same best practices can be applied.
+If yes, use this component with the understanding that:
 
-!!! Insight
+1. Structure will be deprecated/sunsetted in the future
+2. Not all Figma components are built in code
+3. There is no longer a support structure in place if errors/bugs arise
 
-Whatever method you choose to use, the most important thing is maintaining a consistent naming strategy across projects, files, and components.
+If no, or having future support is a must: design and build your own local element using Helios foundations (color, typography, elevation, etc) as a starting point.
+
+### Local components
+
+Components and patterns that are unique to your product should be designed and built locally, extending Helios components and foundations.
+
+![Local component patterns](/assets/getting-started/designers/local-component-patterns.png)
+
+<LinkTo class="doc-link-generic" @route="patterns">Patterns</LinkTo> and guidance around combining and extending Helios components are not currently supported, but are on our [roadmap](https://go.hashi.co/hds-rollout).
 
+!!! Insight
+
+When creating your own library of local components to use in your projects, it's helpful to separate components from your main design work in their own Figma file. This simplifies publishing and creates a single consistent dependency for your design files. For more information on creating and publishing libraries, visit [Figma's official documentation.](https://www.figma.com/best-practices/components-styles-and-shared-libraries/organizing-and-creating-libraries/)
 !!!
 
-#### Teams, projects, files, and pages
+When design and building local components, consider [Atomic Design](https://bradfrost.com/blog/post/atomic-web-design/) methods whenever possible. Helios components generally correspond with the `atom` and `molecule` levels, with some more complex components being considered `organisms`.
+
+## Component instances & detaching
```

Comment:
Question: Should we encode ampersands?

ie, use: `&amp;`

Reply:
That's a good question, I'll surface in the channel

Reply:
After researching further, it seems it should be ok as long as the ampersand has a space before and after (and in some other cases but this is the most common and easy to check)